pkg/check/pushsync/pushsync.go

```diff
@@ -73,25 +70,16 @@ func (c *Check) Run(ctx context.Context, cluster *bee.Cluster, opts interface{})
 	case "light-chunks":
 		return checkLightChunks(ctx, cluster, o)
 	default:
-		return defaultCheck(ctx, cluster, o)
+		return c.defaultCheck(ctx, cluster, o)
 	}
 }
 
 // defaultCheck uploads given chunks on cluster and checks pushsync ability of the cluster
-func defaultCheck(ctx context.Context, c *bee.Cluster, o Options) error {
+func (chk *Check) defaultCheck(ctx context.Context, c *bee.Cluster, o Options) error {
```

Comment:
The receiver name should be consistent `chk` -> `c`. Maybe rename `bee.Cluster` `c` -> `bc`.